features/data/305.Big_Data_Spark/create-spark-jar.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# FIXME -- Just as testing environment
+set > /tmp/fooox
+
+
+mvn install:install-file \
+  -Dfile=./orion.spark.connector-1.2.2.jar \
```

Comment:
Q: Is it the jar as it is described in the tutorial? 

I say it because in Flink, the tutorial makes reference to a previous version of this jar file, therefore, there is an error in the tutorial to the current version of the compiled jar.

Secondly, in Flink this jar is generated. Is it not the case in this tutorial?

Reply:
Scerpt from documentation:

cd cosmos-examples
curl -LO https://github.com/ging/fiware-cosmos-orion-spark-connector/releases/download/FIWARE_7.9.1/orion.spark.connector-1.2.2.jar
mvn install:install-file \
  -Dfile=./orion.spark.connector-1.2.2.jar \
  -DgroupId=org.fiware.cosmos \
  -DartifactId=orion.spark.connector \
  -Dversion=1.2.2 \
  -Dpackaging=jar

mvn package

Reply:
Anyway I correct the script which has some problems like "chown jicg.jicg"

Reply:
But I still see this chown...

Reply:
I removed and pushed again.

Reply:
NTC